src/instrument/devices/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""
+local, custom Device definitions
+"""
+
+# flake8: noqa
+
+## Data management device
```

Comment:
OK here as-is.  For the cookiecutter, we should only do this as directed by content in `iconfig`.